examples/nlp/language_modeling/conf/megatron_gpt_prompt_learning_config.yaml

```diff
@@ -90,6 +90,14 @@ model:
     truncate_field: null
     answer_only_loss: True
     answer_field: "answer"
+  
+  - taskname: "squad" 
```

Comment:
I think we should switch to just having squad in the example config, instead of the other 3 tasks. But we should probably do that in a different PR.